support/node-health-check/src/main.rs

```diff
@@ -0,0 +1,107 @@
+use std::convert::Infallible;
+use std::sync::atomic::AtomicBool;
+use std::sync::atomic::Ordering;
+use std::{sync::Arc, time::Duration};
+
+use anyhow::Result;
+use reqwest::StatusCode;
+use serde::{Deserialize, Serialize};
+use structopt::StructOpt;
+use warp::Filter;
+
+/// Health struct returned by the RPC
```

Comment:
Since these are doc comments, I think they're meant to end with punctuation. I could be wrong though.

Reply:
I searched a bit, but could not find what you mean with the punctuation. I will leave this one open.